src/commands/Info/commandstat.js

```diff
@@ -0,0 +1,18 @@
+exports.run = async function (bot, msg, args) {
+    let command = bot.commands.findBy('name', args[0]);
```

Comment:
You should also check the number of args before trying to fetch the command with a possibly undefined value.